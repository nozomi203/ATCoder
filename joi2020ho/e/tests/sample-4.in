10 10
3 1 4 1 5 9 2 6 5 3
7 1 6
7 8 10
7 2 7
7 3 3
7 1 10
7 2 8
7 1 9
7 4 5
7 7 9
7 10 10
