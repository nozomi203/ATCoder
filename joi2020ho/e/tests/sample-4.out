28
27
34
4
64
43
55
9
27
9
